audio/audio.go

```diff
@@ -35,9 +35,9 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"github.com/hajimehoshi/ebiten/v2/internal/sync"
```

Comment:
Could you move the import path `github.com/hajimehoshi/ebiten...` to a different below group for consistency?